docs/3.0-migration-guide.d/remove_support_for_tls_1.0_1.1_and_dtls_1.0.md

```diff
@@ -3,9 +3,25 @@ Remove suport for TLS 1.0, 1.1 and DTLS 1.0
 
 This change affects users of the TLS 1.0, 1.1 and DTLS 1.0 protocols.
 
-The versions of (D)TLS that are being removed are not as secure as the latest
-versions. Keeping them in the library creates opportunities for misconfiguration
+These versions have been deprecated by RFC 8996.
+Keeping them in the library creates opportunities for misconfiguration
 and possibly downgrade attacks. More generally, more code means a larger attack
 surface, even if the code is supposedly not used.
 
 The migration path is to adopt the latest versions of the protocol.
+
+As a consequence of removing 1.0, support for CBC record splitting was also
+removed, as it was a work-around for a weakness in this particular version.
+There is no migration path is no longer makes sense with newer versions.
```

Comment:
```suggestion
There is no migration path as it no longer makes sense with newer versions.
```
or
```suggestion
There is no migration path since the feature is no longer relevant.
```